spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonAutoConfiguration.java

```diff
@@ -389,6 +395,7 @@ public SimpleQueryBus queryBus(Configuration axonConfiguration, TransactionManag
                                      () -> LoggingQueryInvocationErrorHandler.builder().build()
                              ))
                              .queryUpdateEmitter(axonConfiguration.getComponent(QueryUpdateEmitter.class))
+                             .spanFactory(axonConfiguration.getComponent(SpanFactory.class))
```

Comment:
You can use `configuration.spanFactory()` here:
```suggestion
                             .spanFactory(axonConfiguration.spanFactory())
```

Reply:
A search revealed many more, adjusted them all.